client/app/telecom/telephony/alias/special/fees/telecom-telephony-alias-special-fees.html

```diff
@@ -0,0 +1,147 @@
+<section class="telephony-group-alias-special-fees">
+
+    <header>
+        <section-back-link section-back-link-to-state="telecom.telephony.alias.special"
+                           section-back-link-title="{{ 'telephony_alias_special_fees_title_back_link' | translate }}">
+        </section-back-link>
+        <h1 data-translate="telephony_alias_special_fees_title"></h1>
+    </header>
+
+
+    <div class="telecom-section-content">
+
+        <toast-message></toast-message>
+
+        <div class="row">
+            <div class="col-md-6 widget-presentation"
+                 data-ng-if="!AliasSpecialFeesCtrl.isLoading">
+                <table class="table table-condensed">
```

Comment:
I'm not quite sure, but `table-condensed` doesn't have effect since we use `ovh-ui-kit-bs`.

Reply:
It have little effect at this moment, lifting up and left the row text